Reneiski_Homework1/ReneiskiSelenium11/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Utilities;
+
+namespace ReneiskiSelenium11.PageObjects
+{
+    public class BasePage : Utility
+    {
+        public BasePage(IWebDriver _driver) : base(_driver)
+        {
+        }
+
+        public string GetPageUrl(string pageUrl)
```

Comment:
please implement testsettings and store urls in it rather then pass as a parameter

Reply:
Fixed